spec/features/01_visitors_see_list_of_memes_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+feature 'vistor sees a list of memes' do
+    scenario 'sees a list of memes and link for new meme' do
+
+      ancient_aliens = Meme.create(name: 'Ancient Aliens', image_url: 'http://i3.kym-cdn.com/photos/images/newsfeed/000/158/326/9148130.jpg', description: 'I don\'t know, therefore aliens.')
+      baby_godfather = Meme.create(name: 'Baby Godfather', image_url: 'http://i2.kym-cdn.com/photos/images/newsfeed/000/174/143/finding-nemo-blanket.jpeg')
```

Comment:
Again, you have "useless" assignments here. You dont need to set these creations to variables unless you plan on using them! Otherwise, nix the variables and/or add FactoryGirl